Draft introduction to science of sex and gender (#1408)
ihmeuw/vivarium_research: docs/source/model_design/general_reference_material/bliss/index.rst

**1. This work is important**

Inclusive language use is important, and we want to begin by acknowledging the 

> potential harm that using simplistic and gendered language can create.

Author: I reworked this section, perhaps a bit too much, because I moved some of the content into the intro section below to avoid redundancy.

journey, this guide is intended to be a clear, helpful resource.  

We’re not going to put any words into other people’s mouths, but we will use our own voice to do 
the work and use the terms we find most accurate and inclusive.

1.2 Introduction to sex and gender
----------------------------------

Most human fetuses have either two X chromosomes or one X and one Y chromosome.
A fetus' configuration of X and Y chromosomes influences a number of aspects of development,

> such as hormones, gonads, and genitalia.

Reviewer: While we're on the topic, I wonder if it's worthwhile to introduce the concept of intersex individuals with XXY chromosomes in this section too.

Author: Good idea -- I think [this line](https://github.com/ihmeuw/vivarium_research/pull/1408/files#diff-2542571cb3236fbfd765c3d6fd598eeb8336b9fa314f14547437260ff9e82f7fR206-R207) could be a good place to include examples of the complex combinations of genotype and phenotype that are possible.

Then the child is born into a society with lots of pre-existing ideas about how bodies should
look and work, how people should behave, and how all these things should relate to one another.

> Almost immediately, expectations and norms about identity and behavior are placed on the child,

Reviewer 2: Some are even in place before birth, based on genetic testing!  No need to make this even more complicated, though.

Author: It's funny that you pointed this out because in an earlier version I had noted this.

There is even research showing that [parents begin treating their children differently in the womb](https://www.jstor.org/stable/23799108). But as Abie said I was trying to streamline this paragraph.

based on their external genitalia at birth.
As that child grows up and throughout their life, societal pressures and norms impact their identity and behavior,
and their identity, behavior, and experiences have impacts in turn,
including on very "biological" factors such as hormone levels.
Meanwhile, their chromosomal makeup, hormones, etc. continue to have evolving impacts on their body,
which in turn can affect their identity and behavior, and how society views them.

This constellation of attributes is what we mean by "sex and gender."
It is an incredibly broad umbrella, spanning everything from
chromosomes to clothing styles to power structures.
Despite this range, it is difficult to tease apart the causal impacts of these factors from
one other, because all these factors are strongly correlated with each other.

The dominant culture in the West has historically seen sex and gender as synonymous and
binary.

> According to the traditional view, a person is born either male or female,

Reviewer: ```suggestion
According to this traditional view, a person is born either male or female,
```

> and people born male should identify and behave like boys and men,
> while people born female should identify and behave like girls and women.

Reviewer: ```suggestion
and it is generally assumed that people born male will identify and behave like boys and men,
while people born female will identify and behave like girls and women.
```
Slight preference to remove "should" here to be a bit more neutral, but curious if others have different opinions.

Author: Does this also apply to these lines above?

> Then the child is born into a society with lots of pre-existing ideas about how bodies should
look and work, how people should behave, and how all these things should relate to one another.

I think your edit highlights the ways that societal pressures are created by well-meaning people, which I like. I don't want to totally eliminate reference to the more intentional and conscious ways gender is enforced, though.

Reviewer 3: True - I like how the word 'should' is a moral indicator - it signifies a negative consequence for those who do not identify/behave as expected.

Reviewer: What is we put quotation marks around the word should just to make it more clear that we are not suggesting that this is how people actually "should" behave to the quick reader skimming through the section without full context -- I think that was most of my concern here.

Author: I like that! I'll do it in both places.

who identify with a gender discordant with the sex assigned to them at birth.
Terms like "intersex" and "non-binary" have also gained wider recognition, raising counter-examples to formerly
assumed binaries of both sex and gender.

Recently, scholars have highlighted that sex and gender are **intertwined** and each **multi-dimensional**.

**Intertwined**: Just because there is a biological effect, like a disparity in deaths from cardiovascular disease,
does not mean we should rule out psychosocial causes such as care-seeking behavior or provider
perceptions of pain levels.
Sex and gender characteristics are a tangle of causal relationships and feedback loops. [Ritz_and_Greaves_2022]_

**Multi-dimensional**: Sex includes dimensions such as chromosomes, gonads, hormones, etc, while gender
includes dimensions of identity, presentation, perception, and societal roles.
To say that a person has "male sex" or "female gender" can be a useful shorthand
when most or all of the dimensions usually "align," but it ignores the complexities
of what happens when these dimensions don't align -- and nearly every imaginable

> combination occurs at least some the time.

Reviewer 3: ```suggestion
combination occurs at least some of the time.
```

Furthermore, stopping at observing a difference
between groups categorized like this does little to explain the underlying mechanism.
Many research questions can be more precisely stated in terms of a specific dimension or dimensions
of sex and/or gender. [Bauer_2022]_

It is important to distinguish conceptually between sex and gender,
and where possible between the different dimensions within each of them,

> both to be inclusive of all people and to produce solid research.

Reviewer 3: ```suggestion
both to be inclusive of all people and to produce valid research.
```

However, it is also important not to assume that the causal relationships between
sex- and gender-related factors are simple or easy to determine.

There is a lot we still don't know about the science of sex and gender,
and lots of ongoing research about these topics.
However, this understanding of sex and gender as an intertwined collection
of factors spanning biology, psychology, and sociology is the foundation

> we build on in this guide.

Reviewer 3: ```suggestion
upon which we build in this guide.
```
This just feels slightly more legible to me!
